packages/web-client/app/components/card-pay/create-merchant-workflow/prepaid-card-choice/index.hbs

```diff
@@ -1,23 +1,23 @@
 <ActionCardContainer
   class="prepaid-card-choice"
-  @header="Business Account"
+  @header="Payment profile"
```

Comment:
This is `Payment Profile` elsewhere… does it matter? 🤔

Reply:
It might matter if we change the formatting of text in ActionCardContainer. Will change